openstack/dcs/v2/instances/requests.go

```diff
@@ -272,3 +272,34 @@ func GetConfigurations(c *golangsdk.ServiceClient, instanceID string) (*Configur
 	}
 	return nil, err
 }
+
+type SslOpts struct {
+	Enable *bool `json:"enabled" required:"true"`
+}
+
+func UpdateSsl(client *golangsdk.ServiceClient, id string, opts SslOpts) (*golangsdk.Result, error) {
+	b, err := golangsdk.BuildRequestBody(opts, "")
+	if err != nil {
+		return nil, err
+	}
+
+	var r golangsdk.Result
+	_, err = client.Put(sslURL(client, id), b, &r.Body, &golangsdk.RequestOpts{
+		MoreHeaders: RequestOpts.MoreHeaders,
+	})
+	return &r, err
+}
+
+func GetSsl(client *golangsdk.ServiceClient, id string) (*GetSslResponse, error) {
+	var rst golangsdk.Result
+	_, err := client.Get(sslURL(client, id), &rst.Body, &golangsdk.RequestOpts{
+		OkCodes:     []int{200, 204},
```

Comment:
why can we get 204 status code for GET method?

Reply:
the api returns 204 when no data